Views/GICommitRewriterViewController.m

```diff
@@ -159,7 +159,7 @@ - (BOOL)startRewritingCommit:(GCHistoryCommit*)commit error:(NSError**)error {
   [self.repository notifyRepositoryChanged];
   
   _targetCommit = commit;
-  _titleTextField.stringValue = [NSString stringWithFormat:@"\"%@\" <%@>", _targetCommit.summary, _targetCommit.shortSHA1];
```

Comment:
This should be changed into NSLocalizedString() in a separate and prior commit to make it clear it's a UI string.